src/Avalonia.Visuals/Avalonia.Visuals.csproj

```diff
@@ -3,6 +3,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <RootNamespace>Avalonia</RootNamespace>
+    <LangVersion>latest</LangVersion>
```

Comment:
Isn't there a common build props file you can add it to? I think the same language version should be used throughout the entire solution.

Reply:
Ah yeah good point!